build/commands/lib/util.js

```diff
@@ -25,26 +25,32 @@ async function applyPatches() {
   const patchesPath = path.join(coreRepoPath, 'patches')
   const v8PatchesPath = path.join(patchesPath, 'v8')
   const catapultPatchesPath = path.join(patchesPath, 'third_party', 'catapult')
+  const devtoolsFrontendPatchesPath = path.join(patchesPath, 'third_party', 'devtools-frontend', 'src')
 
   const chromiumRepoPath = config.srcDir
   const v8RepoPath = path.join(chromiumRepoPath, 'v8')
   const catapultRepoPath = path.join(chromiumRepoPath, 'third_party', 'catapult')
+  const devtoolsFrontendRepoPath = path.join(chromiumRepoPath, 'third_party', 'devtools-frontend', 'src')
 
   const chromiumPatcher = new GitPatcher(patchesPath, chromiumRepoPath)
   const v8Patcher = new GitPatcher(v8PatchesPath, v8RepoPath)
   const catapultPatcher = new GitPatcher(catapultPatchesPath, catapultRepoPath)
+  const devtoolsFrontendPatcher = new GitPatcher(devtoolsFrontendPatchesPath, devtoolsFrontendRepoPath)
 
   const chromiumPatchStatus = await chromiumPatcher.applyPatches()
   const v8PatchStatus = await v8Patcher.applyPatches()
   const catapultPatchStatus = await catapultPatcher.applyPatches()
+  const devtoolsFrontendPatchStatus = await devtoolsFrontendPatcher.applyPatches()
 
   // Log status for all patches
   // Differentiate entries for logging
   v8PatchStatus.forEach(s => s.path = path.join('v8', s.path))
   catapultPatchStatus.forEach(
     s => s.path = path.join('third_party', 'catapult', s.path))
+  devtoolsFrontendPatchStatus.forEach(
+    s => s.path = path.join('third_party', 'devtools-frontend', 'src', s.path))
   const allPatchStatus =
-    chromiumPatchStatus.concat(v8PatchStatus).concat(catapultPatchStatus)
+    chromiumPatchStatus.concat(v8PatchStatus).concat(catapultPatchStatus).concat(devtoolsFrontendPatchStatus)
```

Comment:
Maybe it's time to change this to
```js
const allPatchStatus = [
  ...v8PatchStatus,
  ...catapultPatchStatus,
  ...devtoolsFrontendPatchStatus
]

Reply:
Cool! Done and added ChromiumPatcheStatus too.